WtWSMS/common/culture/name_lists/BP_latin.txt

```diff
@@ -1,84 +1,148 @@
-			"dynn_Probianus"
-			"dynn_Sividius"
-			"dynn_Dynamius"
-			"dynn_Glabrio"
-			"dynn_Agapitus"
-			"dynn_Saturninus"
-			"dynn_Valerius"
-			"dynn_Audax"
-			"dynn_Asellus"
-			"dynn_Asterius"
-			"dynn_Viator"
-			"dynn_Corvinus"
-			"dynn_Anicetus"
-			"dynn_Noricorum"
-			"dynn_Marcellinus"
-		}
+		"dynn_Acilius"
+		"dynn_Afranius"
+		"dynn_Septimus_Horatius"
+		"dynn_Lucius_Crixus"
+		"dynn_Populus_Civicus"
+		"dynn_Pompus_Salvius"
+		"dynn_Hadrius_Dextus"
+		"dynn_Egnatius_Drusa"
+		"dynn_Ianuarius_Livius"
+		"dynn_Aurulius_Crixus"
+		"dynn_Lartius"
+		"dynn_Cloelius"
+		"dynn_Potitius"
+		"dynn_Marcius"
+		"dynn_Sempronius"
+		"dynn_Asclepiodotus"
+		"dynn_Aurelianus"
+		"dynn_Symmachus"
+		"dynn_Decius"
+		"dynn_Dracontius"
+		"dynn_Felix"
+		"dynn_Nepos"
+		"dynn_Crassus"
+		"dynn_Eugenius"
+		"dynn_Anicius"
+		"dynn_Iulius_Valerius"
+		"dynn_Libius"
+		"dynn_Olybrius"
+		"dynn_Maximus"
+		"dynn_Egnatius"
+		"dynn_Signus"
+		"dynn_Africanus"
+		"dynn_Modestus"
+		"dynn_Rufinus"
+		"dynn_Drusus"
+		"dynn_Philagrius"
+		"dynn_Canis"
+		"dynn_Quintus"
+		"dynn_Athenaeus"
+		"dynn_Calpurnius"
+		"dynn_Aelius"
+		"dynn_Cornelius"
+		"dynn_Sabinus"
+		"dynn_Pedanius"
+		"dynn_Ducenius"
+		"dynn_Aelianus"
+		"dynn_Pegasus"
+		"dynn_Clemens"
+		"dynn_Fulvus"
+		"dynn_Rutilius"
+		"dynn_Baebius"
+		"dynn_Versus"
+		"dynn_Liberius"
+		"dynn_Decius"
+		"dynn_Cassiodorus"
+		"dynn_Lucimius"
+		"dynn_Reparatus"
+		"dynn_Aconius"
+		"dynn_Ecclesius"
+		"dynn_Glabrio"
+		"dynn_Agapitus"
+		"dynn_Saturninus"
+		"dynn_Valerius"
+		"dynn_Castalius"
+		"dynn_Turcius"
+		"dynn_Viator"
+		"dynn_Rufius"
+		"dynn_Anicetus"
+		"dynn_Noricorum"
+		"dynn_Marcellinus"
+		"dynn_Farsuleius"
+		"dynn_Lentidius"
+		"dynn_Strabonius"
+		"dynn_Tussidius"
+		"dynn_Lurius"
+		"dynn_Rufinius"
+		"dynn_Iuncius"
+		"dynn_Peducaeus"
+		"dynn_Lucceius"
+		"dynn_Sepunius"
+		"dynn_Resius"
+		"dynn_Iustius"
+		"dynn_Orbicius"
+		"dynn_Cavarasius"
+		"dynn_Iucundius"
+		"dynn_Papinius"
+		"dynn_Catienus"
+		"dynn_Cosconius"
+		"dynn_Albinovanus"
+		"dynn_Salvidius"
+		"dynn_Ampudius"
+		"dynn_Scaevinius"
+		"dynn_Nasidius"
+		"dynn_Poppaeus"
+		"dynn_Satrienus"
+		"dynn_Pomptinus"
+		"dynn_Quirinius"
+		"dynn_Ofanius"
+		"dynn_Patulcius"
+		"dynn_Terentilius"
+		"dynn_Scoedius"
+		"dynn_Tapsennius"
+		"dynn_Mescinius"
+		"dynn_Romilius"
+		"dynn_Pinnius"
+		"dynn_Nautius"
+		"dynn_Truttedius"
+		"dynn_Balonius"
+		"dynn_Matrinius"
+		"dynn_Tettidius"
+		"dynn_Umbrenus"	
+		"dynn_Bellienus"
+		"dynn_Sanquinius"
+		"dynn_Crassicius"
+		"dynn_Lemonius"
+		"dynn_Mutius"
+		"dynn_Postumulenus"
+		"dynn_Mimesius"	
+		"dynn_Pacilius"
+		"dynn_Iteius"
+		"dynn_Ianius"
+		"dynn_Acerronius"
+		"dynn_Caepasius"
+		"dynn_Lafrenius"
+		"dynn_Iasdius"
+		"dynn_Lucienus"
+		"dynn_Rullius"
+		"dynn_Spurillius"
+		"dynn_Titanius"
+		"dynn_Umbonius"
+		"dynn_Pescennius"
+		"dynn_Trebellienus"
+		"dynn_Matinius"
+		"dynn_Allienus"
+		"dynn_Istacidius"
+		"dynn_Lacerius"
+		"dynn_Pleminius"
+	}
```

Comment:
Why is the cadet list no longer the same as the dynasty one? I'm a bit confused

Reply:
> Why is the cadet list no longer the same as the dynasty one? I'm a bit confused

Fixed, that was an oversight.